test/e2e/rendererList.ts

```diff
@@ -0,0 +1 @@
+export const renderers = ['WikimediaDesktop', 'VisualEditor']
```

Comment:
This render list should somehow be already available on the code base. I don't see why it should be redefined... or do you want to test less than what is available?

Reply:
We don't keep a list of renderers in a separate place (for now). Only `type renderName` in **abstract.renderer.ts**

Reply:
We do in `sanitize_forceRender()`. Should be defined somewhere where each pice of software can access it, maybe in `const.ts`?

Reply:
I've replaced the list of renderers to `const.ts` as you suggested.